src/database/migrations/20240801074543-merge-mentor-user-extensions.js

```diff
@@ -0,0 +1,155 @@
+'use strict'
+require('module-alias/register')
+require('dotenv').config()
+const materializedViewsService = require('@generics/materializedViews')
+
+module.exports = {
+	up: async (queryInterface, Sequelize) => {
+		await queryInterface.sequelize.query('DROP MATERIALIZED VIEW IF EXISTS m_user_extensions;')
+		await queryInterface.sequelize.query('DROP MATERIALIZED VIEW IF EXISTS m_mentor_extensions;')
+
+		await queryInterface.sequelize.transaction(async (transaction) => {
+			// Add the is_mentor flag to user_extensions
+			await queryInterface.addColumn(
+				'user_extensions',
+				'is_mentor',
+				{
+					type: Sequelize.BOOLEAN,
+					allowNull: false,
+					defaultValue: false,
+				},
+				{ transaction }
+			)
+
+			// Copy data from mentor_extensions to user_extensions
+			await queryInterface.sequelize.query(
+				`
+        INSERT INTO user_extensions (
```

Comment:
@nevil-mathew the user is mentee then he become mentor, in this case this statement will create duplicate entry in user_extensions ?